ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml

```diff
@@ -1,7 +1,89 @@
 periodics:
 - agent: kubernetes
   cluster: vsphere
-  cron: 0 0 * * 2,4
+  cron: 0 0 * * 3
```

Comment:
nothing changed on 4.13, but the order of the jobs were sorted to the newest release first